aws_lambda_builders/workflows/go_modules/workflow.py

```diff
@@ -0,0 +1,51 @@
+"""
+Go Modules Workflow
+"""
+from aws_lambda_builders.workflow import BaseWorkflow, Capability
+
+from .actions import GoModulesBuildAction
+from .builder import GoModulesBuilder
+from .validator import GoRuntimeValidator
+from .utils import OSUtils
+
+
+class GoModulesWorkflow(BaseWorkflow):
+
+    NAME = "GoModulesBuilder"
+
+    CAPABILITY = Capability(language="go",
+                            dependency_manager="modules",
+                            application_framework=None)
+
+    def __init__(self,
+                 source_dir,
+                 artifacts_dir,
+                 scratch_dir,
+                 manifest_path,
+                 runtime=None,
+                 osutils=None,
+                 **kwargs):
+
+        super(GoModulesWorkflow, self).__init__(
+            source_dir,
+            artifacts_dir,
+            scratch_dir,
+            manifest_path,
```

Comment:
Just making sure I understand correctly. `go build` doesn't need to have `go.mod` file passed into the command. So that is the reason why this workflow doesn't need to use the manifest_path?

Reply:
Right, `go build` walks up the directory tree to look for `go.mod`, no need to pass it in.